src/components/NcPopover/NcPopover.vue

```diff
@@ -121,6 +121,36 @@ export default {
 			type: Boolean,
 			default: true,
 		},
+
+		delay: {
+			type: Number,
+			default: 0,
+		},
+
+		handleResize: {
+			type: Boolean,
+			default: true,
+		},
+
+		shown: {
+			type: Boolean,
+			default: false,
+		},
+
+		placement: {
+			type: String,
+			default: 'auto',
+		},
+
+		boundary: {
+			type: String,
```

Comment:
Floating vue expects a DOM element. This triggers an invalid prop type check now.
```
[Vue warn]: Invalid prop: type check failed for prop "boundary". Expected String, got HTMLBodyElement 

found in

---> <NcPopover> at src/components/NcPopover/NcPopover.vue
       <NcActions> at src/components/NcActions/NcActions.vue
         <TaskBody> at src/components/TaskBody.vue
           <Draggable>
             <TaskDragContainer> at src/components/TaskDragContainer.vue
               <General> at src/views/AppContent/General.vue
                 <Collections> at src/views/AppContent/Collections.vue
                   <NcAppContent> at src/components/NcAppContent/NcAppContent.vue
                     <NcContent> at src/components/NcContent/NcContent.vue
                       <App> at src/App.vue
                         <Root> tasks-main.js:260533:21
```